docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -28,6 +28,9 @@ support:
   - Required region name for S3.
 * - `s3.path-style-access`
   - Use path-style access for all requests to S3
+* - `s3.canned-acl`
+  - Amazon S3 supports a set of predefined grants, known as canned ACLs.
```

Comment:
Let's copy the documentation from the legacy S3 documentation, with some changes:
```
  - [Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl)
    to use when uploading files to S3. Defaults to `NONE`, which has the same
    effect as `PRIVATE`. If the files are to be uploaded to an S3 bucket owned
    by a different AWS user, the canned ACL may be set to one of the following:
    `PRIVATE`, `PUBLIC_READ`, `PUBLIC_READ_WRITE`, `AUTHENTICATED_READ`,
    `BUCKET_OWNER_READ`, or `BUCKET_OWNER_FULL_CONTROL`.